resolwe/flow/management/commands/clean_test_dir.py

```diff
@@ -0,0 +1,53 @@
+""".. Ignore pydocstyle D400.
+
+====================
+Clean test directory
+====================
+
+Command to run on local machine::
+
+    ./manage.py clean_test_dir
+
+"""
+import re
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from resolwe.flow.managers.protocol import ExecutorFiles
+
+TEST_DIR_REGEX = r'test_\d+'
+
+
+class Command(BaseCommand):
+    """Cleanup files created during testng."""
+
+    help = "Cleanup files created during testng."
+
+    def handle(self, *args, **kwargs):
+        """Cleanup files created during testng."""
+
+        for name in ['DATA_DIR', 'UPLOAD_DIR', 'RUNTIME_DIR']:
+            directory = os.path.abspath(settings.FLOW_EXECUTOR[name])
```

Comment:
[`FLOW_EXECUTOR_SETTINGS`](https://github.com/genialis/resolwe/blob/master/resolwe/test/testcases/setting_overrides.py#L10) should be used here.